ark/type/__tests__/keywords/string.test.ts

```diff
@@ -43,6 +43,17 @@ contextualize(() => {
 		attest(b64url("fn5+").toString()).equals(
 			'must be base64url-encoded (was "fn5+")'
 		)
+
+		const b64parse = type("string.base64.parse")
+		attest(Buffer.from(b64parse("fn5+") as Uint8Array).toString("utf8")).snap(
```

Comment:
Ideally snap would recognize `Uint8Array` as a builtin here and not map over it, but seems like you'd still have to convert it to a string to easily snapshot it. This seems fine to me for now unless you want to make a change to snapshotting in attest to add builtin logic for this.